apm-sniffer/apm-sdk-plugin/mysql-2.x-plugin/src/main/java/org/skywalking/apm/plugin/jdbc/mysql/vo/StatementEnhanceRequireCacheObject.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017, OpenSkywalking Organization All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Project repository: https://github.com/OpenSkywalking/skywalking
+ */
+
+package org.skywalking.apm.plugin.jdbc.mysql.vo;
+
+import org.skywalking.apm.plugin.jdbc.trace.ConnectionInfo;
+
+/**
+ * {@link StatementEnhanceRequireCacheObject} contain the {@link org.skywalking.apm.plugin.jdbc.trace.ConnectionInfo}
+ * and <code>sql</code> for enhance.
+ *
+ * @author zhangxin
+ */
+public class StatementEnhanceRequireCacheObject {
```

Comment:
I don't agree `StatementEnhanceRequireCacheObject` is a good name. These is nothing about `cache` first, every JavaBean is about keeping some values, and you can't call every JavaBean as `*CacheObject`. I think the name should be `StatementMetricNameKeyInfos`.

And you should add new package ` org.skywalking.apm.plugin.jdbc.mysql.vo` for it. Because only one class in a separated package is a strange thing, specially unlikely you are not going to add more classes in that package. Right?